code/modules/projectiles/guns/projectile/shotgun/slidebarrelshotgun.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/weapon/gun/projectile/shotgun/slidebarrelshotgun
+	name = "Slide Barrel Shotgun"
+	desc = "Slide Barrel Shotgun, made out of trash, but rather special on its design"
+	icon = 'icons/obj/guns/projectile/sawnshotgun.dmi'
+	icon_state = "sawnshotgun"
+	item_state = "sawnshotgun"
+	max_shells = 1
+	caliber = CAL_SHOTGUN
+	handle_casings = HOLD_CASINGS
+	slot_flags = SLOT_BELT|SLOT_HOLSTER
+	can_dual = 1
+	ammo_type = /obj/item/ammo_casing/shotgun/pellet
+	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
+	w_class = ITEM_SIZE_NORMAL
+	force = WEAPON_FORCE_PAINFUL
+	damage_multiplier = 0.8 //same as sawn-off
+	recoil_buildup = 1.2 //gonna have solid grip on those, point-blank shots adviced
```

Comment:
This is probably too low. To be fair the sawn-off also has its recoil set too low.

Reply:
oh, wow.. most have 14, and the pump got.. 1.2 that is indeed.. akward.. and low.. well making the slide 16